superset/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -82,6 +79,30 @@ export function changeFilterValue(filter, value) {
   return { type: CHANGE_FILTER_VALUE, filter, value };
 }
 
-export function setFieldValue(key, value) {
-  return { type: SET_FIELD_VALUE, key, value };
+export function setFieldValue(key, value, label) {
+  return { type: SET_FIELD_VALUE, key, value, label };
+}
+
+export const SAVE_SLICE_SUCCEEDED = 'SAVE_SLICE_SUCCEEDED';
```

Comment:
do we need `SAVE_SLICE_STARTED` to show feedback to the user while we are saving?